sunpy/map/sources/psp.py

```diff
@@ -0,0 +1,43 @@
+"""
+Solar Orbiter Map subclass definitions.
+"""
+
+from sunpy.map import GenericMap
+
+__all__ = ['WISPRMap']
+
+
+class WISPRMap(GenericMap):
+    """
+    WISPR Map
+
+    The The Wide-field Imager for Parker Solar Probe (WISOR) is a white light
+    telescope onboard the Parker Solar Probe (PSP) spacecraft.
+
+    References
+    ----------
+    * `PSP science gateway <https://sppgway.jhuapl.edu//>`__
+    * `WISPR Instrument Page <https://wispr.nrl.navy.mil//>`__
+    * `Instrument Paper <https://doi.org/10.1007/s11214-014-0114-y>`__
+    """
+    @property
+    def processing_level(self):
+        lvl = self.meta.get('level', None)
+        if lvl is None:
+            return
+        return int(lvl[1])
+
+    @property
+    def exposure_time(self):
+        return self.meta.get('xposure', 0.0) * self.timeunit
+
```

Comment:
WISPR consists of two telescopes/detectors Inner and Outer.
In the fits header:
DETECTOR == 1 is for INNER telescope fits files, and
DETECTOR == 2 is for Outer.
I think it would worth adding a "detector" property here, right?

Reply:
This already works through `GenericMap` - I'll add a test for it though 👍

Reply:
Also in STEREO HIMap there is this property:
@property
def measurement(self):
        """
        Returns the type of data observed.
        """
        # TODO: This needs to do more than white-light.  Should give B, pB, etc.
        return "white-light"

do we need this here as well?

Reply:
The WISPR data guide (https://wispr.nrl.navy.mil/sites/wispr.nrl.navy.mil/files/wispr_data_user_guide_v3.pdf) shows slightly different bandpasses for WISPR-I and WISPR-O, but both are referred to as white light according to the in-flight calibration paper https://link.springer.com/article/10.1007%2Fs11207-021-01847-9 .